projects/WTI-UI/src/app/modules/clarifications/components/clarifications-page/clarifications-page.component.ts

```diff
@@ -50,14 +52,17 @@ export class ClarificationsPageComponent implements OnInit, OnDestroy {
       data: {},
       disableClose: true
     });
+
   }
 
   private filterClarifications() {
     const fitlerParams = this.filterForm.value;
     let filtered = this.clarifications;
-    if (fitlerParams.receipient === 'all') { filtered = filtered.filter(x => x.recipient === 'All'); }
-    if (fitlerParams.receipient === 'team') { filtered = filtered.filter(x => x.recipient !== 'All'); }
-    if (fitlerParams.problem) { filtered = filtered.filter(x => x.problem === fitlerParams.problem); }
+
+
+    if (fitlerParams.receipient === 'all') { filtered = filtered.filter(x => x.recipient === 'All'); console.log('receipient is ALl');}
+    if (fitlerParams.receipient === 'team') { filtered = filtered.filter(x => x.recipient !== 'All'); console.log('receipient is a team')}
+    if (fitlerParams.problem) { filtered = filtered.filter(x => x.problem === fitlerParams.problem); console.log('receipient is a problem')}
```

Comment:
Two comments about the changes in the above three lines:
- I don't think the console logging is necessary any longer; if you agree then I suggest removing it.
- These changes conflict with related changes proposed in PR #1041.  That's fine; collisions are to be expected when multiple people are working in similar areas of the code.  I just note it here so we'll remember that the conflicts will need to be resolved in whichever PR is merged later.
  - (If you do remove the console logging, then I think these lines will no longer conflict?)

Reply:
I agree, the console logging should be removed.